drivers/census_tracking/app/models/census_tracking/worksheet.rb

```diff
@@ -0,0 +1,90 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module CensusTracking
+  class Worksheet
+    include ArelHelper
+
+    def initialize(filter)
+      @filter = filter
+    end
+
+    def projects
+      projects = GrdaWarehouse::Hud::Project.
+        preload(:organization).
+        viewable_by(@filter.user).
+        where(id: @filter.effective_project_ids)
+
+      projects.map { |project| [HUD.project_type(project.compute_project_type), project.organization.name, project.safe_project_name, project.id] }.
```

Comment:
I _think_ this should use computed_project_type, which you can get right of the db.